src/dispatch/static/dispatch/src/case/DetailsTab.vue

```diff
@@ -29,6 +29,14 @@
           />
         </ValidationProvider>
       </v-flex>
+      <v-flex xs6>
+        <v-select
+          v-model="resolution_reason"
+          label="Resolution Reason"
+          :items="visibilities"
```

Comment:
`resolutionReasons` instead?

Reply:
That doesn't map to what the api returns.

Reply:
I don't understand. You're providing visibilities not resolution reasons defined in https://github.com/Netflix/dispatch/pull/3090/files#diff-15e369cbda358d580577a6be233ec92377ef358ce0797560ff0f221e263fd47aR161 as an options to select from?

Reply:
<img width="601" alt="Screenshot 2023-03-10 at 1 47 00 PM" src="https://user-images.githubusercontent.com/39573146/224434595-4dfcf9fd-aee9-4b6a-b280-f0841ebb555c.png">